layers/+email/mu4e/config.el

```diff
@@ -15,5 +15,8 @@
 (defvar mu4e-account-alist nil
   "Account alist for custom multi-account compose.")
 
+(defvar mu4e-async nil
```

Comment:
rename it `mu4e-enable-async-operations` and be more explicit about what it does in the docstring (for now only sending message asynchronously)